homeassistant/components/deconz/.translations/bg.json

```diff
@@ -9,7 +9,7 @@
             "no_key": "\u041d\u0435 \u043c\u043e\u0436\u0430 \u0434\u0430 \u0441\u0435 \u043f\u043e\u043b\u0443\u0447\u0438 API \u043a\u043b\u044e\u0447"
         },
         "step": {
-            "init": {
+            "user": {
```

Comment:
this won't work, the trnalsations are stored in lokalise.co

We can either keep the step being called `init`, or rename the keys on lokalise, or have all translators translate it again (which is a dick move)